server/channels/api4/access_control.go

```diff
@@ -777,3 +779,49 @@ func convertToVisualAST(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Logger.Warn("Error while writing response", mlog.Err(err))
 	}
 }
+
+func createAccessControlSyncJob(c *Context, w http.ResponseWriter, r *http.Request) {
+	var jobData map[string]string
+	if jsonErr := json.NewDecoder(r.Body).Decode(&jobData); jsonErr != nil {
+		c.SetInvalidParamWithErr("job_data", jsonErr)
+		return
+	}
+
+	auditRec := c.MakeAuditRecord(model.AuditEventCreateJob, model.AuditStatusFail)
+	defer c.LogAuditRec(auditRec)
+	model.AddEventParameterToAuditRec(auditRec, "job_type", model.JobTypeAccessControlSync)
+	model.AddEventParameterToAuditRec(auditRec, "job_data", jobData)
```

Comment:
Back to my comment of job data being "anything the user want to add", I wonder if we should "clean it" here too. 0/5